packages/deployments/contracts/tasks/addConnectors.ts

```diff
@@ -26,7 +27,7 @@ export default task("add-connectors", "Add all connectors to the root manager")
     const deployer = Wallet.fromMnemonic((networkConfig.accounts as any).mnemonic as unknown as string);
 
     const env = mustGetEnv(_env);
-    const remove = _remove ?? true;
+    const remove = _remove ? _remove === "true" : true;
```

Comment:
instead of having this line, just change
```
.addOptionalParam("remove", "Whether or not to remove connectors that exist")
```
to
```
.addFlag("remove", "Whether or not to remove connectors that exist")
```

that way we can just use it as a flag, like:
```
ywc run hardhat addConnectors --remove
```